truckersmp_cli/variables.py

```diff
@@ -51,6 +51,7 @@ class File:
     """Files."""
 
     loginvdf_inner = "config/loginusers.vdf"
+    steamlibvdf_inner = "steamapps/libraryfolders.vdf"
```

Comment:
Beware that older installations may still have `SteamApps`.

Valve changed that for new installations to all lowercase some time ago but I'm not sure if they transitioned existing installations.